zomes/chat/src/lib.rs

```diff
@@ -111,10 +111,11 @@ fn init(_: ()) -> ExternResult<InitCallbackResult> {
 
 #[hdk_extern]
 fn genesis_self_check(data: GenesisSelfCheckData) -> ExternResult<ValidateCallbackResult> {
-    if validation::skip_proof_sb(data.dna_def.properties) {
+    if validation::skip_proof_sb(data.dna_def.properties.clone()) {
```

Comment:
Whenever I  find myself adding a clone, I take a moment to consider whether I could pass a borrow instead. In this case it seems that both `skip_proof_sb` and `holo_agent`  could take a reference to the properties and avoid this cloning